layouts/partials/webinar-list-item.html

```diff
@@ -0,0 +1,21 @@
+<li class="w-full md:w-1/2 lg:w-1/3 m-0 p-2">
+    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col mx-auto" style="max-width: 320px;">
+        <div style="height: 239px;">
+            <div class="w-full h-full bg-contain bg-no-repeat bg-center bg-purple-500" style="background-image: url('{{ .Params.preview_image }}');"></div>
```

Comment:
Why using `background-image` and not an `<img>`?

Reply:
It worked better for displaying the different types of images. One is a preview from youtube and the other is a png of a Google Slide I created. Using `background-image` made the positioning a bit easier for me.

I filed an issue to create preview images for all the webinar/videos here: https://github.com/pulumi/docs/issues/2668